tests/unit/template_cassdc_test.go

```diff
@@ -130,5 +133,42 @@ var _ = Describe("Verify CassandraDatacenter template", func() {
 			// Two containers, medusa and cassandra
 			Expect(len(cassdc.Spec.PodTemplateSpec.Spec.Containers)).To(Equal(2))
 		})
+
+		It("setting allowMultipleNodesPerWorker to true", func() {
```

Comment:
Can you add two more unit tests - one where `allowMultipleNodesPerWorker` is `true` and `resources` are not specified and one where `allowMultipleNodesPerWorker` is `false` and `resources` is set?